toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -146,11 +165,12 @@ func buildCosiFile(sourceDir string, outputFile string, partitions []outputParti
 	}
 
 	metadata := MetadataJson{
-		Version:   "1.0",
-		OsArch:    getArchitectureForCosi(),
-		Id:        imageUuidStr,
-		Images:    make([]Image, len(imageData)),
-		OsRelease: osRelease,
+		Version:    "1.1",
```

Comment:
let's keep this at 1.1 until the entire spec is implemented, otherwise we'll have some compatibility issues